app/Http/Resources/StreamResource.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class StreamResource extends JsonResource
+{
+    public function toArray($request): array
+    {
+        return [
+            'id' => $this->youtube_id,
```

Comment:
I think `id` is not the correct wording for this field, please use `platform_id` instead since we might add Twitch streams too.

Reply:
Not a problem! I used `id` here to follow JSON:API specification as each resource needs it's own identifier. I would recommend keeping `id` here but perhaps adding UUIDs to the model? Then withing attributes you can add:

```php
'identifiers' => [
    'youtube' => $this->youtube_id,
    'twitch' => $this->twitch_id,
]
```

Reply:
Good point. Yeah, guess it would be good then to add UUIDs. Could you add that field and also make sure all given streams get a UUID through the migration?

Reply:
For now I have set this to ID of the model